lib/cluster/idCardHandler.ts

```diff
@@ -216,7 +216,11 @@ export class ClusterIdCardHandler {
 
     const exitHandler = () => {
       if (!childProcess.killed || childProcess.connected) {
-        childProcess.kill();
+        try {
+          childProcess.disconnect();
+        } catch (e) {
+          // It could happens that the worker has been killed before the dispose causing disconnect to fail
```

Comment:
no necessary to add a debug log at least here instead of silent ?

Reply:
No not really, even though the process fails to close the connection for some reason, the call to process.exit will stop the process and then the OS is going to close the IPC between them, leading the child process to stop himself and clear the ID Card inside Redis